Core/Strict.hs

```diff
@@ -0,0 +1,37 @@
+-- |
+-- Module      : Core.Strict
+-- License     : BSD-style
+-- Maintainer  : Foundatinon
+-- Stability   : stable
+-- Portability : portable
+--
+-- Enforce strictness when executing lambda
+--
+
+module Core.Strict
+    ( ($!)
+    , strict2
+    , strict3
+    , strict4
+    , strict5
+    , strict6
+    ) where
+
+($!) :: (a -> b) -> a -> b
+($!) f !a = f a
```

Comment:
ah ok, did you mean "remove"? should I do a git revert on it?

On 24 July 2016 at 18:12, Nicolas DI PRIMA <notifications@github.com> wrote:

> In Core/Strict.hs
> <https://github.com/haskell-foundation/foundation/pull/49#discussion_r71991168>
> :
>
> > +-- Portability : portable
> > +--
> > +-- Enforce strictness when executing lambda
> > +--
> > +
> > +module Core.Strict
> > +    ( ($!)
> > +    , strict2
> > +    , strict3
> > +    , strict4
> > +    , strict5
> > +    , strict6
> > +    ) where
> > +
> > +($!) :: (a -> b) -> a -> b
> > +($!) f !a = f a
>
> ok, my bad, this will conflict with Prelude's one. Can you remote it?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/haskell-foundation/foundation/pull/49/files/56b57b3eb2b5bd9f3e9ca551fd1ead97ffe21013#r71991168>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACj1rvE8gheq6RgwyxN0jqBRYRDuUZLGks5qY47bgaJpZM4JTkbx>
> .
>